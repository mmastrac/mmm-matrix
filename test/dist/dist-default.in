export INPUT_INPUT="[]"
export INPUT_DEFAULT="[default: true]"
