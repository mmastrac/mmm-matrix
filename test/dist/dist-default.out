::group::Input matrix
[]

::endgroup::
::group::Default matrix
- default: true

::endgroup::
::group::Config object
null

::endgroup::
No matrix items were generated, using default
::group::Generated matrix
- default: true

::endgroup::

::set-output name=matrix::[{"default":true}]
Success.
